apps/cli/src/utils.ts

```diff
@@ -1,6 +1,24 @@
 import { dirname } from "node:path";
 import { access, mkdir, writeFile, constants, rm } from "node:fs/promises";
 
+export interface File {
+  name: string;
+  content: string;
+  encoding: "utf-8";
+}
+
+export interface Folder {
+  name: string;
+  files: File[];
+  subFolders: Folder[];
+}
+
+export enum ProjectType {
```

Comment:
we don't use enums at all :)

TS enums is a mistake, they are hiding whats behind them and TS team already admitted it was a mistake. They try to not introduce runtime code any more.